frontend/src/scenes/experiments/secondaryMetricsLogic.ts

```diff
@@ -65,9 +65,14 @@ export const secondaryMetricsLogic = kea<secondaryMetricsLogicType>([
     actions({
         // modal
         openModalToCreateSecondaryMetric: true,
-        openModalToEditSecondaryMetric: (metric: SecondaryExperimentMetric, metricIdx: number) => ({
+        openModalToEditSecondaryMetric: (
+            metric: SecondaryExperimentMetric,
+            metricIdx: number,
+            readOnly: boolean = false
```

Comment:
Instead of readOnly I think better to have a `showResults` param, because its not read only version of the metric selector, but a completely different insight - to avoid confusion